app/helpers/organisation_helper.rb

```diff
@@ -1,9 +1,5 @@
 module OrganisationHelper
   def app_name
-    organisation_name.presence || "Open HQ"
-  end
-
-  def organisation_name
-    Metadata.get("organisation_name")
```

Comment:
Got rid of metadata stuff as right now we don’t need it and when we do we can probably use redis to store key value settings.